_strategies/human-centered-design.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: strategy
+title: Human-Centered Design
+description: Build a product that works well for your users.
+permalink: /human-centered-design/
+tags: strategy
+image: /assets/img/strategy-images/hcd.jpg
+image-credit: https://www.flickr.com/photos/nearsoft/
+---
+
+## What is human-centered design and why is it important in modular contracting?
+
+Human centered design is an approach to engage the end user in software development to yield actionable, prioritized, and assumption-challenging results. By directly engaging users throughout, HCD builds products that work better for users, require less rework, and prompt fewer support calls. We believe that theses approaches need to be incorporated at every step of the agile process. From conducting design research before and at the beginning of an acquistition, and usability testing for every sprint and user story developed.
```

Comment:
I don't think the last sentence is a complete sentence. Suggest "For example, teams can conduct design research before and at the beginning of an acquisition, and can conduct usability testing during every development sprint."